docker-for-mac/release-notes.md

```diff
@@ -20,6 +22,64 @@ Desktop for Mac](install.md#download-docker-for-mac).
 
 ## Stable Releases of 2019
 
+## Docker Desktop Community 2.1.0.0
+2019-07-31
+
+[Download](https://hub.docker.com/?overlay=onboarding)
```

Comment:
Previously we were putting `[Download](https://download.docker.com/mac/stable/36874/Docker.dmg)` here, otherwise you will need to update this link when pushing the next release, and dig some history to retrieve the right build number at this time.

Reply:
@gtardif - Discussed this with @nebuk89. We can discuss the use cases in detail tomorrow and replace the Hub links with specific URLs if required.